validator/validators/performance/performance_validator.go

```diff
@@ -213,3 +321,15 @@ func isAllValuesGreaterThanOrEqualToZero(values []float64) bool {
 	}
 	return true
 }
+
+func isAllStatisticsGreaterThanOrEqualToZero(datapoints []types.Datapoint) bool {
+	if len(datapoints) == 0 {
+		return false
```

Comment:
If `datapoints` is empty then it's actually true that all statistics are non-negative since there are no data points that are negative. Is there a reason it needs to return `false` in this case?

Reply:
If datapoints is empty then that means we did not get any metric which should throw an error.